pkg/vadmin/interface.go

```diff
@@ -84,6 +85,10 @@ type Dispatcher interface {
 	// cluster quorum. The IP given for each vnode *must* match the current IP
 	// in the vertica catalog. If they aren't a call to ReIP is necessary.
 	StartDB(ctx context.Context, opts ...startdb.Option) (ctrl.Result, error)
+
+	// InstallPackages will install all packages under /opt/vertica/packages
+	// where Autoinstall is marked true.
+	InstallPackages(ctx context.Context, opts ...installpackages.Option) error
```

Comment:
As discussed on Teams, it might be good to return the per package install status. That way the operator could surface errors if some packages had errors during install. We can opt to defer this to a follow-on PR too if you feel this is too much to add to this one.

Reply:
will do this in this PR